docs_src/getting-started/index.md

```diff
@@ -7,22 +7,16 @@ Developer/Contributor.
 
 If you want to get the EdgeX platform and run it (but do not
 intend to change or add to the existing code base now) then you
-are considered a "User". You will want to follow the
-[Getting Started Users](./Ch-GettingStartedUsers.md) guide. The
-Getting Started Users guide takes you through the process of getting
-the latest release EdgeX Docker Containers from Docker Hub. If you wish
-to get the latest EdgeX containers (those built from the current ongoing
-development efforts prior to release), then see
-[Getting Started Users - Nexus](./Ch-GettingStartedUsersNexus.md). 
-
-!!! WARNING
-    Containers used from Nexus are considered "work in progress". There is no guarantee
-    that these containers will function properly or function properly with
-    other containers from the current release.
```

Comment:
These are moved to `Ch-GettingStartedCDevelopers.md` and `Ch-GettingStartedUsersNexus.md` pages.